week1/index.js

```diff
@@ -1,25 +1,61 @@
-// To run this file: node index.js
-// Then in your browser: http://localhost:8080
-var http = require('http');
+// Then in your browser: http://localhost:8000
+let http = require('http');
 
-var port = 8080;
+//I added this package to deal with files.
+let fs = require('fs');
 
-var server = http.createServer();
+//I think this is not the best way to add style file, but it a good example to use 'fs' pakage.
+let style = fs.readFileSync("style.css")
+
+// This is the magic number :-)
+const port = 8000;
+let state = 10;
+let server = http.createServer();
+let info;
 
 // Start the HTTP server, start listening for requests
-server.listen(port, function(error) {
-  if (error) {
-    console.log(error);
-  } else {
-    console.log('api listening on port', port);
-  }
+server.listen(port, function (error) {
+	if (error) {
+		console.log(error);
+	} else {
+		console.log('api listening on port', port);
+	}
 });
 
-// Create a event handler for "request"
-// this is an alternative way
-server.on('request', function(request, response) {
-  console.log('New http request received', request.url);
-  response.setHeader('content-type', 'text/html');
-  response.write('<html><head></head><body><h1>Hello world</h1></body></html>');
-  response.end();
-});
\ No newline at end of file
+// Create a event handler for "request" and calling render function. 
+
+server.on('request', function (request, response) {
+	response.setHeader('content-type', 'text/html');
+	switch (request.url) {
+		case '/':
+			info = 'Create HTML server';
```

Comment:
Your use of `info` as a global variable could be problematic (take a look at the comments I left for Nour in Slack and on his first pull request). What will happen if two requests come in at the exact same time? (Hint - this is called a race condition)